src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class User {
+    private String name;
+    private int age;
+    private List<Bet> userBets;
+
+    public User(String name, int age) {
+        this.name = name;
+        this.age = age;
+        userBets = age >= 21 ? new ArrayList<>() : null;
```

Comment:
if `userBets` is null, what happens when you call the method on it? (it happens in your app multiple times without a check)

I would recommend to keep it simple and not to concentrate on business logic at all at this point

Reply:
done